pandas/io/excel/_base.py

```diff
@@ -79,7 +79,7 @@
     )
 _read_excel_doc = (
     """
-Read an Excel file into a pandas DataFrame.
+Read an Excel file into a ``pandas`` ``DataFrame``.
```

Comment:
I'm thinking we shouldn't wrap `DataFrame` throughout. In my opinion, it doesn't need the highlighting, and it can serve as a distraction.

Reply:
Good call.  I wasn't sure on that one so I'm glad you provided some clarity on it.  It could go either way, but I agree that it gets distracting with how often DataFrame is referenced in the docs.